tests/test_fakes.py

```diff
@@ -938,12 +947,20 @@ def test_show_tables(dcur: snowflake.connector.cursor.SnowflakeCursor):
             "kind": "TABLE",
             "database_name": "DB1",
             "schema_name": "SCHEMA1",
+            "comment": None,
```

Comment:
similar to objects, `terse schemas` doesn't have the comment column
```suggestion
```

Reply:
gotcha. I know for all three of the new ones, `terse` has no effect